packages/library-base/src/commonMain/kotlin/io/realm/notifications/RealmChangeImpl.kt

```diff
@@ -14,12 +14,10 @@
  * limitations under the License.
  */
 
-package io.realm
+package io.realm.notifications
 
-/**
- * A `callback` interface to receive notifications about updates.
- */
-internal fun interface Callback<T> {
-    fun onChange(result: T)
-    // FIXME API-NOTIFICATION Consider adding an onError(throwable: Throwable)
-}
+import io.realm.BaseRealm
+
+internal class InitialRealmImpl<R : BaseRealm>(override val realm: R) : InitialRealm<R>
```

Comment:
These should be moved to `io.realm.internal` or `io.realm.internal.notifications` since `internal` classes are turned public when consumed from Java

Reply:
Neat! It has to be `io.realm.internal.notifications` because it has to be on the same package.